src/frequenz/sdk/microgrid/client.py

```diff
@@ -532,7 +565,12 @@ async def set_power(self, component_id: int, power_w: int) -> Empty:
             )
         return result
 
-    async def set_bounds(self, component_id: int, lower: float, upper: float) -> None:
+    async def set_bounds(
+        self,
+        component_id: int,
+        lower: float,
+        upper: float,
+    ) -> None:
```

Comment:
there also seem to be a number of formatting changes that didn't get cleaned up.

Reply:
All checks passed so I assumed everything is fine.